lib/deaf_grandma.rb

```diff
@@ -21,7 +21,14 @@ def run!
   def speak(input)
 
     #Implement your code here <<<<<<<<<
+    @bye_counter += 1  if (input.include? "BYE")
```

Comment:
while your use of parenthenses works here in this method, its not the ruby way. in some cases you don't need them at all and in other places you should change how they're implemented.

https://github.com/bbatsov/ruby-style-guide